acceptance_tests/README.md

```diff
@@ -0,0 +1,10 @@
+BRP Acceptance Tests
+
+Install:
+
+Bundle install the Gem file 

```

Comment:
So it appears that I need to install the gems separately with something like `gem install cucumber` or using bundler, otherwise the Gemfile fails with the error mentioned above. What would you propose?
We'll need to run these tests in CI, so we'll need a set of commands to tell the CI env to run after installation.